jax/_src/cudnn/scaled_matmul_stablehlo.py

```diff
@@ -639,6 +643,14 @@ def scaled_dot_bwd(dimension_numbers, preferred_element_type, configs, res, g):
   }
   grad_lhs = scaled_dot_general_transpose_lhs(*args, **lhs_kw_args)
   grad_rhs = scaled_dot_general_transpose_rhs(*args, **rhs_kw_args)
+
+  if configs[2].mode == "nvfp4":
```

Comment:
Can we add some comments here? like
```
We apply a Straight-Through Estimator (STE) with zero-out behavior: if inputs are clipped during quantization in fprop, their corresponding gradients are zeroed out; otherwise, they pass through unchanged.
```